Cargo.toml

```
@@ -16,7 +16,7 @@ readme = "README.md"
 libavif-sys = { version = "0.17.0", path = "libavif-sys", default-features = false }
 
 [dev-dependencies]
-image = "0.25"
+image = {version="0.25", default-features=false, features=["png"]}
```

I remember I left all features enabled so that the examples would work with any format.

If we want to change it let's fix the fmt:

```suggestion
image = { version = "0.25", default-features = false, features = ["png"] }
```

---

webp requires a newer MSRV, and it's also webp. Maybe we can only add jpeg to the list of features?

---

Having jpeg and png should be fine :+1: